modules/rosa-how-upgrades-work.adoc

```diff
@@ -0,0 +1,44 @@
+
+// Module included in the following assemblies:
+//
+// upgrading/rosa-upgrading-sts.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="rosa-how-upgrades-work_{context}"]
+= How ROSA (classic architecture) cluster upgrades work
+
+Upgrades are manually initiated (one-time) or automatically scheduled (recurring). Red{nbsp}Hat Site Reliability Engineers (SREs) monitor upgrade progress and either proactively notify you to take corrective actions or remedy issues encountered.
+
+The Cluster Version Operator (CVO) is the primary component that orchestrates and facilitates the OpenShift Container Platform update process. The scheduling, monitoring, and notifications of ROSA (classic architecture) cluster upgrades are orchestrated through the Managed Upgrade Operator (MUO). The MUO manages the orchestration of automated in-place cluster upgrades by satisfying the operating conditions that a managed cluster must hold before and after initiating the cluster upgrade.
+
+[id="rosa-upgrade-scheduled-time_{context}"]
+== Cluster upgrade scheduled time
+
+You can schedule cluster upgrades by setting the scheduled time. This is when the preparation for the cluster upgrade begins with pre-upgrade health checks and additional compute capacity creation. The actual cluster upgrade starts within one hour from the scheduled time. You receive an email notification when the cluster upgrade starts.
+
+You can observe the status of the cluster upgrade by running the `oc get clusterversion` command in the ROSA CLI (`rosa`).
+
+[id="rosa-cluster-upgrade-overview_{context}"]
+== ROSA (classic architecture) upgrade overview
+
+The following high-level steps that occur during the ROSA (classic architecture)cluster update:
+
+. Before the cluster upgrade begins, a cluster health check is performed by the  MUO. If the MUO identifies an issue that requires corrective action, you will be notified. Some examples of the cluster health checks that MUO performs include:
```

Comment:
We can add one more high level step for when upgrade is scheduled in advance which would trigger the PreHealthCheck and notify user of any failures which can be addressed before the upgrade begins.